src/main/webapp/src/components/Navbar.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  menuButton: {
+    marginRight: theme.spacing(2),
+  },
+  title: {
+    flexGrow: 1,
+  },
+}));
+
+export default function Navbar() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <AppBar style={{ background: "#733F94" }} position="static">
+        <Toolbar>
+          <Typography variant="h6" className={classes.title}>
+            Alpollo
+          </Typography>
+          <Button color="inherit">Search</Button>
+          <Button color="inherit">Login</Button>
```

Comment:
Probably the authentication is going to be linked here... Maybe sync with @sashapff to implement it?